app/main.py

```diff
@@ -1,3 +1,86 @@
-def shop_trip():
-    # write your code here
-    pass
+import json
+from os import path
+
+from app.customer import Customer
+from app.shop import Shop
+
+
+def shop_trip() -> None:
+    _path = path.join("app", "config.json")
+    with open(_path, "r") as file:
+        data_from_file = json.load(file)
+        try:
+
+            fuel_price = data_from_file["FUEL_PRICE"]
+
+        except KeyError as e:
+            print(e, "not exist")
```

Comment:
The error message 'not exist' for missing keys in the JSON data is not very informative. Consider providing more context in the error message, such as which key is missing, to make debugging easier.